pkg/events/events.go

```diff
@@ -0,0 +1,120 @@
+package events
+
```

Comment:
I think we should rename this file to _handler.go_.

Reply:
You have a much better sense of making uniform names for packages/modules than I do :-) thanks